### PR TITLE
Make distroless images default in docs

### PR DESCRIPTION
**Pull request by tcsc**

### CODE REVIEW

File: docs/config.json [hunk 1]
```diff
@@ -1467,8 +1467,10 @@
         "version": "12.1.1"
       },
       "helm_repo_url": "https://charts.releases.teleport.dev",
-      "latest_oss_docker_image": "public.ecr.aws/gravitational/teleport:12.1.1",
-      "latest_ent_docker_image": "public.ecr.aws/gravitational/teleport-ent:12.1.1"
+      "latest_oss_docker_image": "public.ecr.aws/gravitational/teleport-distroless:12.1.5",
+      "latest_oss_debug_docker_image": "public.ecr.aws/gravitational/teleport-distroless-debug:12.1.5",
+      "latest_ent_docker_image": "public.ecr.aws/gravitational/teleport-ent-distroless:12.1.5",
+      "latest_ent_debug_docker_image": "public.ecr.aws/gravitational/teleport-distroless-debug:12.1.5"
```
Comment (r0mant) [inline docs/config.json:1473]: Typo?
```suggestion
      "latest_ent_debug_docker_image": "public.ecr.aws/gravitational/teleport-ent-distroless-debug:12.1.5"
```

**Review by r0mant — APPROVED** [review]

File: e
Comment (marcoandredinis) [inline e:1]: This seems to have a lot of changes :thinking:

File: docs/pages/includes/docker-images-oss.mdx [hunk 2]
```diff
@@ -10,9 +10,17 @@ version of Teleport from the three most recent release branches. They are
 stable, and we recommend their use to keep your Teleport installation up to
 date.
 
-|Image name|Teleport version|Image automatically updated?|Image base|
+|Image name|Teleport version|Image automatically updated?|Troubleshooting Tools?|Image base|
```
Comment (ptgott) [inline docs/pages/includes/docker-images-oss.mdx:13]: It looks like the table isn't rendering properly on the dev server (`yarn dev`):

Screenshot 2023-04-04 at 6 04 03 PM

File: docs/pages/includes/docker-images-oss.mdx [hunk 3]
```diff
@@ -10,9 +10,17 @@ version of Teleport from the three most recent release branches. They are
...
+|`public.ecr.aws/gravitational/teleport-distroless:(=teleport.major_version=)`|The latest version of Teleport Open Source|Yes|No|[Distroless Debian 11](https://github.com/GoogleContainerTools/distroless)|
+|`public.ecr.aws/gravitational/teleport-distroless-debug:(=teleport.major_version=)`|The latest version of Teleport Open Source|Yes|Yes|[Distroless Debian 11](https://github.com/GoogleContainerTools/distroless)|
+|`(=teleport.latest_oss_docker_image=)`|The version specified in the image's tag (i.e. (=teleport.version=))|No|No|[Distroless Debian 11](https://github.com/GoogleContainerTools/distroless)|
+|`(=teleport.latest_oss_debug_docker_image=)`|The version specified in the image's tag (i.e. (=teleport.version=))|No|Yes|[Distroless Debian 11](https://github.com/GoogleContainerTools/distroless)|
 
 For testing, we always recommend that you use the latest released version of Teleport, which is currently `(=teleport.latest_oss_docker_image=)`.
+
+The `*-debug` images are intended for troubleshooting deployments, and are not indended for production use.
```
Comment (ptgott) [inline docs/pages/includes/docker-images-oss.mdx:22]: How do these images allow for troubleshooting?

File: docs/pages/includes/enterprise/docker-images.mdx [hunk 4]
```diff
@@ -4,14 +4,23 @@ automatically rebuilt every night.
...
+These images are hosted on our [Amazon ECR Public repository](https://gallery.ecr.aws/gravitational/teleport-ent-distroless). All tags under `public.ecr.aws/gravitational/teleport-ent-distroless` are Teleport Enterprise images. Each tag points to a multi-arch image, conmtaining Teleport for `arm`, `arm64` and `amd64`.
 
-| Image name | Open Source or Enterprise? | Teleport version | Image automatically updated? | Image base |
+FIPS images are still posted to our [Legacy Amazon ECR Public repository](https://gallery.ecr.aws/gravitational/teleport-ent).
+You can specify the architecture of a Teleport FIPS Docker image by adding a postfix to the image tag, e.g., `public.ecr.aws/gravitational/teleport:(=teleport.major_version=)-arm64`. Versions without an architecture postfix use the `amd64` architecture, e.g., `public.ecr.aws/gravitational/teleport:(=teleport.major_version=)`. 
+
+| Image name | Open Source or Enterprise? | Teleport version | Image automatically updated? | Includes troubleshooting tools | Image base |
```
Comment (ptgott) [inline docs/pages/includes/enterprise/docker-images.mdx:12]: This table also isn't rendering on the dev server

image

File: docs/pages/includes/enterprise/docker-images.mdx [hunk 5]
```diff
@@ -4,14 +4,23 @@ automatically rebuilt every night.
...
+| `(=teleport.latest_ent_docker_image=)-fips` | Enterprise FIPS | The version specified in the image's tag (i.e. (=teleport.version=)) | No | Yes | [Ubuntu 20.04](https://hub.docker.com/\_/ubuntu) |
+
+For testing, we always recommend that you use the latest release version of Teleport Enterprise, which is currently `(=teleport.latest_ent_docker_image=)`.
+
+[Ubuntu 20.04](https://hub.docker.com/\_/ubuntu)-based images for non-FIPS 
+Teleport are available from our [Legacy Amazon ECR Public repository](https://gallery.ecr.aws/gravitational/teleport-ent).
 
-For testing, we always recommend that you use the latest release version of Teleport Enterprise, which is currently `(=teleport.latest_ent_docker_image=)`.
\ No newline at end of file
+The `*-debug` images are intended for troubleshooting deployments, and are not indended for production use.
```
Comment (ptgott) [inline docs/pages/includes/enterprise/docker-images.mdx:26]: Same comment re: the `*-debug` images

**Comment by ptgott** [comment]
@tcsc Does this need to be backported?

File: docs/pages/includes/docker-images-oss.mdx (hunk 3 above)
Comment (tcsc) [reply docs/pages/includes/docker-images-oss.mdx:22]: The have a shell and busybox

**Review by marcoandredinis — APPROVED** [review]
Aside from current comments, LGTM

File: docs/pages/includes/docker-images-oss.mdx (hunk 2 above)
Comment (tcsc) [reply docs/pages/includes/docker-images-oss.mdx:13]: Fixed

File: docs/pages/includes/enterprise/docker-images.mdx (hunk 4 above)
Comment (tcsc) [reply docs/pages/includes/enterprise/docker-images.mdx:12]: Fixed

File: docs/config.json (hunk 1 above)
Comment (tcsc) [reply docs/config.json]: Fixed

**Comment by tcsc** [comment]
> @tcsc Does this need to be backported?

Yes, but only to v12

File: docs/pages/management/guides/docker.mdx [hunk 6]
```diff
@@ -109,9 +114,8 @@ Move your `license.pem` file to `~/teleport/data`.
 # Generate a sample Teleport config and write it to the local config directory.
 # This container will write the config and immediately exit--this is expected.
 $ docker run --hostname localhost --rm \
-  --entrypoint=/bin/sh \
-  -v ~/teleport/config:/etc/teleport \
-  (=teleport.latest_ent_docker_image=) -c "teleport configure > /etc/teleport/teleport.yaml"
+  --entrypoint=/usr/local/bin/teleport \
+  ${TELEPORT_DOCKER_IMAGE} configure > ~/teleport/config/teleport.yaml
```
Comment (marcoandredinis) [inline docs/pages/management/guides/docker.mdx:118]: Should we change line 124 as well?
It still shows the markdown placeholder and not the env var defined above
```
 (=teleport.latest_ent_docker_image=)
```

**Review by marcoandredinis — APPROVED** [review]

File: docs/pages/management/guides/docker.mdx (hunk 6 above)
Comment (tcsc) [reply docs/pages/management/guides/docker.mdx:118]: Done

**Pull request merged**